tests/utils/testrunner.go

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package utils
+
+import (
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+)
+
+// runnerFailExitCode is the exit code when test runner setup is failed
+const runnerFailExitCode = 1
+
+// testingMInterface interface is used for testing TestRunner
+type testingMInterface interface {
```

Comment:
nit :probably typo, what does 'M' stand for in testingMInterface ?

Reply:
This is the interface for [go testing.M](https://golang.org/pkg/testing/#M) in order to test testrunner. I may need to change the name of interface :) I would rename it to `runnable`

Reply:
Fixed to runnable